core/main/src/Core/Particle.ts

```diff
@@ -408,6 +411,10 @@ export class Particle implements IParticle {
     }
 
     public draw(delta: IDelta): void {
+        if (this.orbitOptions.enable) {
+            this.container.canvas.drawOrbit(this, this.orbitOptions);
```

Comment:
This should be inside the `drawParticle` method

Reply:
Moved to drawParticle.